torch/csrc/lazy/ts_backend/ts_native_functions.cpp

```diff
@@ -430,113 +370,68 @@ at::Tensor & LazyNativeFunctions::normal_(at::Tensor & self, double mean, double
     // return self;
 };
 
-at::Tensor LazyNativeFunctions::permute(const at::Tensor& self,
-                                        at::IntArrayRef dims) {
-  TORCH_LAZY_FN_COUNTER("lazy::");
-  auto self_tensor = torch::lazy::TryGetLtcTensor(self);
-  return torch::lazy::CreateAtenFromLtcTensor(torch::lazy::permute(
-      self_tensor, torch::lazy::ToI64Vector(dims)));
-}
-
-at::Tensor LazyNativeFunctions::select(const at::Tensor& self, int64_t dim,
-                                       int64_t index) {
-  TORCH_LAZY_FN_COUNTER("lazy::");
-  return torch::lazy::CreateAtenFromLtcTensor(
-      torch::lazy::select(torch::lazy::TryGetLtcTensor(self), dim, index));
-}
-
-at::Tensor LazyNativeFunctions::slice(const at::Tensor& self, int64_t dim,
-                                      c10::optional<int64_t> start,
-                                      c10::optional<int64_t> end,
-                                      int64_t step) {
-  int64_t start_val = start.has_value() ? start.value() : 0;
-  int64_t end_val = end.has_value() ? end.value() : INT64_MAX;
+at::Tensor LazyNativeFunctions::_unsafe_view(const at::Tensor& self,
+                                     at::IntArrayRef size) {
   TORCH_LAZY_FN_COUNTER("lazy::");
-  return torch::lazy::CreateAtenFromLtcTensor(torch::lazy::slice(
-      torch::lazy::TryGetLtcTensor(self), dim, start_val, end_val, step));
+  return LazyNativeFunctions::view_copy(self, size);
 }
 
-at::Tensor LazyNativeFunctions::squeeze(const at::Tensor& self) {
-  TORCH_LAZY_FN_COUNTER("lazy::");
-  return torch::lazy::CreateAtenFromLtcTensor(
-      torch::lazy::squeeze(torch::lazy::TryGetLtcTensor(self)));
-}
 
-at::Tensor LazyNativeFunctions::squeeze(const at::Tensor& self, int64_t dim) {
-  TORCH_LAZY_FN_COUNTER("lazy::");
-  return torch::lazy::CreateAtenFromLtcTensor(
-      torch::lazy::squeeze(torch::lazy::TryGetLtcTensor(self), dim));
+// This is needed by the torch.tensor constructor.
+// LazyTensor always opts into functionalization.
+// "lifting" a tensor for functionalization means wrapping it in a FunctionalTensorWrapper object.
+at::Tensor LazyNativeFunctions::lift(const at::Tensor& tensor) {
```

Comment:
oh, is 'lift' a new aten op? or is this just a normal function, added to the LazyNativeFunctions:: namespace which is a little confusing

Reply:
Yep, `at::lift()` is an actual aten op that hooks into the dispatcher. And it's basically a no-op for most cases, except for Lazy/XLA.